tests/src/test/scala/cats/tests/ConstSuite.scala

```diff
@@ -66,6 +67,27 @@ class ConstSuite extends CatsSuite {
     }
   }
 
+  checkAll("Const[String, Int]", FunctorTests[Const[String, ?]].functor[Int, Int, Int])
+  checkAll("Functor[Const[String, ?]]", SerializableTests.serializable(Functor[Const[String, ?]]))
 
+  {
+    implicit val setMonoid: CommutativeMonoid[Int] =
```

Comment:
Should we name this `intCommutativeMonoid` instead, maybe?

Reply:
ops ... fixing it 👍